lib/config/src/app/healthcheck.rs

```diff
@@ -11,8 +11,9 @@ pub enum HealthCheckV1 {
     schemars::JsonSchema, serde::Serialize, serde::Deserialize, PartialEq, Eq, Clone, Debug,
 )]
 pub struct HealthCheckHttpV1 {
-    /// Path to the health check endpoint.
-    pub path: String,
+    #[serde(flatten)]
+    pub request: super::HttpRequest,
```

Comment:
all the other fields of `HttpRequest` are duplicated/redefined here

Reply:
Yep, some duplications left in, those are fixed.